puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/foreman-documentation.yaml

```diff
@@ -0,0 +1,9 @@
+- job:
+    name: foreman-documentation-test
+    node: 'admin && sshkey'
```

Comment:
You don't need this since you're not using any SSH keys

Reply:
Thanks I was not sure.